From 11c966cc630393e322ef6b88df91d16247bbfc37 Mon Sep 17 00:00:00 2001
From: Robbie Harwood <rharwood@redhat.com>
Date: Mon, 11 Mar 2019 14:56:31 -0400
Subject: [PATCH] [tests] Test suite fixes for virtualenv and clang

- Typo fix - VIRTUAL_ENV in magtests.py
- testenv object manipulation fix in magtests.py
- Work around -fstack-clash-protection problems in clang

Signed-off-by: Robbie Harwood <rharwood@redhat.com>
(cherry picked from commit 6aa0a5bfe9d60a50cf36c561268c5d7c1fdb2f0e)
[rharwood@redhat.com: drop Travis goo]
---
 tests/magtests.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tests/magtests.py b/tests/magtests.py
index f14f47a..a4842a0 100755
--- a/tests/magtests.py
+++ b/tests/magtests.py
@@ -687,7 +687,7 @@ if __name__ == '__main__':
 
         # support virtualenv
         testenv['PATH'] = os.environ.get('PATH', '')
-        testenv['ViRTUAL_ENV'] = os.environ.get('VIRTUAL_ENV', '')
+        testenv['VIRTUAL_ENV'] = os.environ.get('VIRTUAL_ENV', '')
 
         testenv['DELEGCCACHE'] = os.path.join(testdir, 'httpd',
                                               USR_NAME + '@' + TESTREALM)
@@ -716,6 +716,9 @@ if __name__ == '__main__':
                    'MAG_USER_NAME_2': USR_NAME_2,
                    'MAG_USER_PASSWORD_2': USR_PWD_2}
         testenv.update(kdcenv)
+        testenv['PATH'] = os.environ.get('PATH', '')
+        testenv['VIRTUAL_ENV'] = os.environ.get('VIRTUAL_ENV', '')
+
         errs += test_basic_auth_krb5(testdir, testenv, logfile)
 
         errs += test_no_negotiate(testdir, testenv, logfile)
